crates/apollo-router/src/router_factory.rs

```diff
@@ -44,4 +62,35 @@ impl RouterFactory<ApolloRouter, ApolloPreparedQuery> for ApolloRouterFactory {
         .map(|res| res.expect("ApolloRouter::new() is infallible; qed"))
         .boxed()
     }
+
+    fn recreate(
+        &self,
+        graph: Arc<ApolloRouter>,
```

Comment:
This thing has been renamed to "router".

```suggestion
        router: Arc<ApolloRouter>,
```

Reply:
done